cli/src/opts/cast.rs

```diff
@@ -30,35 +30,35 @@ pub struct Opts {
 )]
 pub enum Subcommands {
     /// Prints the maximum value of the given integer type.
-    #[clap(name = "--max-int", visible_aliases = &["max-int", "maxi"])]
+    #[clap(name = "max-int", visible_aliases = &["max-int", "maxi"])]
```

Comment:
Oh yeah it's breaking—it's why i'm merging to the v1 branch.

Will add the old name back! the intention was to remove them tho

Reply:
I think it's reasonable to keep them as aliases so this doesn't break

Reply:
Worth noting that “some commands use `--` and some don’t” was inherited from dapptools’ Seth, which had the same thing. I don’t know what the logic was for the difference offhand, and I recall finding it confusing, so I’m supportive of this change. Maybe @tynes has more info or thoughts here

As a side note, cast probably needs an interface cleanup at some point, since it’s kind of a cluttered patchwork of commands right now. We can probably clean things up by grouping various commands behind new subcommands and introducing some renames.